pkg/discovery/dns/miekgdns/resolver.go

```diff
@@ -41,6 +49,14 @@ func (r *Resolver) LookupSRV(ctx context.Context, service, proto, name string) (
 				Priority: addr.Priority,
 				Port:     addr.Port,
 			})
+		case *dns.CNAME:
+			var resp []*net.SRV
```

Comment:
Is there any reason to declare this `var`?

Reply:
Now that I think of it it's redundant :P thought declaring it would be more explicit. I'll delete it 👍

Reply:
I think this is the only remaining nit and the CHANGELOG addition and it should be good :blush:

Reply:
done!